### ISSUE TITLE
Running Vitis AI on Xilinx RF SoC

### ISSUE THREAD

**Issue by udara28** [Xilinx/Vitis_Embedded_Platform_Source#9]
I am wondering whether it will be possible to adopt Zynq MPSoC flow to implement Vitis AI on Xilinx RF SoC platforms like ZCU111 or ZCU1275. These devices also have the same ARM A53 QuadCore PS.
If it is possible what files should I modify to properly port the designs to RF SoC boards?

**Comment by imrickysu** [comment]
Yes, it's possible. A Vitis platform for these boards is required. The Vitis-AI applications don't need changes.
User guide and tutorials of how to create custom Vitis platforms will be published in Aug.

**Comment by udara28** [comment]
Thank you for the reply. I would really appreciate if you could share the relevant links to create custom platforms once they are published.

**Comment by imrickysu** [comment]
Please refer to the platform creation tutorial
https://github.com/Xilinx/Vitis-In-Depth-Tutorial/tree/master/Vitis_Platform_Creation
